Makefile

```diff
@@ -151,19 +151,27 @@ test-ext-dev-e2e: $(OPERATOR_SDK) $(KUSTOMIZE) $(KIND) #HELP Run extension creat
 	test/extension-developer-e2e/setup.sh $(OPERATOR_SDK) $(CONTAINER_RUNTIME) $(KUSTOMIZE) $(KIND) $(KIND_CLUSTER_NAME) $(E2E_REGISTRY_NAMESPACE)
 	go test -count=1 -v ./test/extension-developer-e2e/...
 
-.PHONY: test-unit
 ENVTEST_VERSION := $(shell go list -m k8s.io/client-go | cut -d" " -f2 | sed 's/^v0\.\([[:digit:]]\{1,\}\)\.[[:digit:]]\{1,\}$$/1.\1.x/')
+CONTROLLER_RUNTIME_VERSION := release-$(shell go list -m sigs.k8s.io/controller-runtime | awk '{print $$2}' | sed -E 's/^v([0-9]+)\.([0-9]+)\..*/\1.\2/')
 UNIT_TEST_DIRS := $(shell go list ./... | grep -v /test/)
 COVERAGE_UNIT_DIR := $(ROOT_DIR)/coverage/unit
-test-unit: $(SETUP_ENVTEST) #HELP Run the unit tests
+
+.PHONY: setup-envtest #HELP Install setup-envtest binary in the bin directory.
+setup-envtest:
+	mkdir -p $(ROOT_DIR)/bin
+	GOBIN=$(ROOT_DIR)/bin go install sigs.k8s.io/controller-runtime/tools/setup-envtest@$(CONTROLLER_RUNTIME_VERSION)
```

Comment:
I wouldn't say I liked this bingo thing because I work with other projects, and it replaces the modules and setup of things that are set up in my local environment and have no relation to the project.

The binary in the dir bin ensures that the project can use it and does not affect my local environment. Also, it allows me to download other envtest binaries if I need to.

Reply:
In other words, setup-envtest is a binary and not a GO module to be set in go/bin.

Reply:
IMO this is better off not cluttering the Makefile and left to bingo to handle. Bingo does not change local binaries except for the project.

Reply:
Bingo will setup things under $GOPATH/go/bin
That is valid for ANY Golang project in my local env
So, yes, it affects all my other projects.

Reply:
Bingo includes the version number on the binaries, so projects can share binaries because every project is still able to specify a specific version to use.

The maintainers aligned on bingo over a year ago, and I think it is more important to be consistent on tool usage. If we want to move away from bingo, we should do it consistently, not as a one-off.

Reply:
Hi @joelanford,

This setup is really problematic as it doesn’t respect my local environment.

**A good example of the issue is:**

I encountered a significant problem when trying to run kind load image directly (outside of the project target). The command failed because of a version mismatch between the kind binary installed by Bingo and the version I expected to use.

To make matters worse, if I want to run any kind command, I now have to remember that Bingo overrides my setup. This means I need to use GOPATH/bin/kind_<version> instead of simply running the command from my own installed version.

This behavior is frustrating and doesn’t align with respecting local setups, which is critical for a smooth development experience.

Reply:
Anyway, as it is as it is
I will change it to use the BINGO
and came back to the BINGO staff.